website/content/en/docs/examples/gha.md

```diff
@@ -44,7 +46,6 @@ jobs:
         GITHUB_TOKEN: ${{ github.token }}  # required by `gh attestation verify`
       run: |
         set -eux
-        LIMA_VERSION=$(curl -fsSL https://api.github.com/repos/lima-vm/lima/releases/latest | jq -r .tag_name)
```

Comment:
Can we just write this to `$GITHUB_ENV`?

Reply:
I think so. But I think it's simpler and better  to have the version configured explicitly. Otherwise you could have two builds from the same commit that use different versions of Lima.

Reply:
But it is hard to keep updating the version then (unless dependabot can handle it)

Reply:
Personally, I'd rather not have my CI jobs switch Lima version automatically.

I can see that some might want it to automatically update. However, this is sample code and I think it should lean towards simplicity.

Reply:
Let me know what you decide and I'll revise the PR if necessary.

Reply:
I still prefer the automatic update here.
It is obvious how to pin a specific version and it does not need to be documented here.

Reply:
Ok, I updated and force-pushed.